distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFanout.java

```diff
@@ -14,31 +14,48 @@
 package ddf.test.itests.catalog;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.hasItem;
 import static org.junit.Assert.fail;
+import static com.jayway.restassured.RestAssured.delete;
 import static com.jayway.restassured.RestAssured.get;
 import static com.jayway.restassured.RestAssured.given;
-import static com.jayway.restassured.RestAssured.when;
 
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.xml.xpath.XPathExpressionException;
+
+import org.apache.http.HttpStatus;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.ops4j.pax.exam.junit.PaxExam;
 import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
 import org.ops4j.pax.exam.spi.reactors.PerClass;
 
 import com.jayway.restassured.path.json.JsonPath;
+import com.jayway.restassured.response.Response;
 
 import ddf.common.test.BeforeExam;
 import ddf.test.itests.AbstractIntegrationTest;
+import ddf.test.itests.common.Library;
 
 @RunWith(PaxExam.class)
 @ExamReactorStrategy(PerClass.class)
 public class TestFanout extends AbstractIntegrationTest {
 
     private static final String LOCAL_SOURCE_ID = "ddf.distribution";
 
+    // Using default resource tag as the one to blacklist
+    private static final List<String> TAG_BLACKLIST = Collections.singletonList("resource");
+
+    private static final TestCatalog TEST_CATALOG = new TestCatalog();
```

Comment:
If we are going to be using this functionality across multiple test classes we should extract the common code into something catalog helper.